@navikt/core/react/src/guide-panel/GuidePanel.tsx

```diff
@@ -47,7 +48,10 @@ export const GuidePanel = forwardRef<HTMLDivElement, GuidePanelProps>(
       <div className="navds-guide">
         {illustration ?? <DefaultIllustration />}
       </div>
-      <div className="navds-guide-panel__content">{children}</div>
+      <div className="navds-guide-panel__content">
+        <SpeechBubbleArrow />
```

Comment:
Making the arrow an absolute positioned child of the panel it's "attached" to seemed like the most stable arrangement.

Reply:
I'm really really hoping 🙏  they never "detach" under various scenarios 😬 ... (i get major ick from websites that have "detaching elements" haha... it sinks the percieved quality of the entire site to 0 😓 )